src/components/Education.js

```diff
@@ -1,34 +1,131 @@
 import styles from './education.module.css';
+import { useState, Fragment } from 'react';
+import { format } from 'date-fns';
 
-function handleClick(e) {}
+export default function Education({ data, onChange, onClick }) {
+  const [isEditing, setIsEditing] = useState(false);
+
+  function handleClick() {
+    isEditing ? setIsEditing(false) : setIsEditing(true);
+  }
+
+  function generateContent(schoolData) {
+    let content = (
+      <div className={styles.infoContainer}>
+        <label htmlFor="school">
+          School
+          <div className="school" id="school">
+            {schoolData.school}
+          </div>
+        </label>
+        <label htmlFor="title">
```

Comment:
label usually goes with form elements. I suppose you don't need <label /> when you're not editing and using <input>